plugins/txt/webportal/templates/components/live-scene-txt.hbs

```diff
@@ -0,0 +1 @@
+<button {{action 'addTxt' 'scenePose'}} class="btn btn-default">Send Txt</button>
```

Comment:
You actually don't need the 'scenePose' parameter, because you're not doing anything with it in the live-scene-txt JS file.

Reply:
REALLY? Because LET ME TELL YOU how much effort it took to make this work. I bet I tried a billion things.  

My javascript is so hacky.

Reply:
Hey look, I didn't need that. Fixed.